packages/actor-dereference-file/test/ActorDereferenceFile-test.ts

```diff
@@ -5,9 +5,10 @@ import type { IActorDereferenceOutput } from '@comunica/bus-dereference';
 import { ActorDereference } from '@comunica/bus-dereference';
 import { ActionContext, Bus } from '@comunica/core';
 import type { IActionContext } from '@comunica/types';
-import * as streamToString from 'stream-to-string';
 import { ActorDereferenceFile } from '../lib/ActorDereferenceFile';
 
+const streamToString = require('stream-to-string');
```

Comment:
Could you add the following comment to each location that we replaced with `require`?
`// Use require instead of import for default exports, to be compatible with variants of esModuleInterop in tsconfig.`